src/views/AddLiquidity.vue

```diff
@@ -139,64 +140,13 @@
           {{ t('createPair.supply') }}
         </template>
       </s-button>
-      <slippage-tolerance class="slippage-tolerance-settings" />
+      <add-liquidity-transaction-details
+        v-if="areTokensSelected && isAvailable && (!emptyAssets || (liquidityInfo || {}).balance)"
```

Comment:
I saw this expression in `AddLiquidityTransactionDetails` component. Could it be defined in `TokenPair` mixin?

Reply:
this check is removed at all since it should be shown anyway when its wrapper (collapse component) is rendered